index.html

```diff
@@ -948,30 +948,38 @@ <h2><dfn>Coalesced events</dfn></h2>
             particularly for fast and large movements. Using the
-            <code><a data-lt="PointerEvent.getCoalescedEvents">getCoalescedEvents</a></code> method
+            <code><a data-lt="PointerEvent.getCoalescedEvents">getCoalescedEvents()</a></code> method
             it is possible for applications to access the raw, un-coalesced position changes. These
             allow for a more precise handling of pointer movement data. In the case of drawing
             applications, for instance, the un-coalesced events can be used to draw smoother curves that
             more closely match the actual movement of a pointer.</p>
 
             <figure id="figure_coalesced">
                 <img src="images/coalesced-points.png" alt="Close-up view of a curve, showing coalesced and un-coalesced points">
-                <figcaption>Example of a curve in a drawing application – using only the coalesced
+                <figcaption>Example of a curve in a drawing application — using only the coalesced
                 coordinates from <code>pointermove</code> events (the grey dots), the curve is noticeably
                 angular and jagged; the same line drawn using the more granular points provided by
-                <code>getCoalescedEvents</code> (the red circles) results in a smoother approximation
+                <code>getCoalescedEvents()</code> (the red circles) results in a smoother approximation
                 of the pointer movement.</figcaption>
             </figure>
 
             <p>A <a>PointerEvent</a> has an associated <dfn>coalesced events list</dfn> (a list of
-            zero or more <code>PointerEvent</code>s). If a trusted event is a <code>pointermove</code> or
-            <code>pointerrawupdate</code> event, the list is a sequence of all <code>PointerEvent</code>s
-            that were coalesced into this event; otherwise it is an empty list.
+            zero or more <code>PointerEvent</code>s). For trusted <code>pointermove</code> and
+            <code>pointerrawupdate</code> events, the list is a sequence of all <code>PointerEvent</code>s
+            that were coalesced into this event. For all other trusted event types, it is an empty list.
             Untrusted events have their <a>coalesced events list</a> initialized to the value passed to the
             constructor.</p>
 
-            <p>The events in the coalesced events list of a trusted event will have increasing
-            {{Event/timeStamp}}s, so the first event will have the smallest {{Event/timeStamp}}.</p>
+            <p>The events in the coalesced events list of a trusted event will have:</p>
+            <ul>
+                <li>Monotonically increasing {{Event/timeStamp}} values [[DOM]] — all coalesced events have a
+                    {{Event/timeStamp}} that is smaller than the {{Event/timeStamp}} of the dispatched pointer event that the
```

Comment:
I think this should be not greater than. It may be possible that there could be coalesced events with the same timestamp.

Reply:
so `smaller than, or equal to, the ...` ? or is that too handwavy sounding? (basically `<=`)

Reply:
I think smaller than or equal to is also fine.